lib/cli.js

```diff
@@ -12,7 +12,7 @@ if (process.argv.length !== 3) {
 
 const opts = config({
   root: resolve(process.argv[2]),
-  runBundler: true,
+  bundle: true,
```

Comment:
I think we should manually set watch to false here in cli too. My rationale is for clarity sake. Even though watch defaults to false I think it helps instantly communicate here in the cli command that watching is off.

Reply:
I agree